### ISSUE TITLE
Undo pollution of MethodCallParameter

### ISSUE THREAD

**Issue by snakefoot** [NLog/NLog#2567]
Think it would be best to remove these parameters from `MethodCallParameter`

```
        /// <summary>
        /// Parameter can combine multiple LogEvents into a single parameter value
        /// </summary>
        public bool EnableGroupLayout { get; set; }

        /// <summary>
        /// Group Header when combining multiple LogEvents into a single parameter value (<see cref="EnableGroupLayout"/>)
        /// </summary>
        public Layout GroupHeaderLayout { get; set; }

        /// <summary>
        /// Group Item Separator when combining multiple LogEvents into a single parameter value (<see cref="EnableGroupLayout"/>)
        /// </summary>
        public Layout GroupItemSeparatorLayout { get; set; }

        /// <summary>
        /// Group Footer when combining multiple LogEvents into a single parameter value (<see cref="EnableGroupLayout"/>)
        /// </summary>
        public Layout GroupFooterLayout { get; set; }
```

They were introduced with #2223, but I'm not sure I like this change after reading my code again. Prefer to keep `MethodCallParameter` simple without being involved in this batching/group-logic.

**Comment by snakefoot** [comment]
Alternative solution is adding the following properties to WebServiceTarget:

```
GroupCreator-Layout - Used to perform BucketSort
GroupHeader-Layout
GroupItemSeperator-Layout
GroupFooter-Layout
```

So the layout will become:

```
  GroupHeader
     Parameters for LogEvent1
     GroupItemSeperator
     Parameters for LogEvent2
  GroupFooter
```

Each group will become a separate request.

**Comment by snakefoot** [comment]
Would prefer to have this fixed before NLog 4.5 RTM, but I will not have time before 21th February

**Comment by 304NotModified** [comment]
Good idea
